src/ast.ts

```diff
@@ -3,6 +3,7 @@
 import { parseClass } from './brace-expressions.js'
 import { MinimatchOptions, MMRegExp } from './index.js'
 import { unescape } from './unescape.js'
+import { ArrayPrototypeFilter, ArrayPrototypeJoin, ArrayPrototypeMap, ArrayPrototypePop, ArrayPrototypePush, ArrayPrototypeSlice, ObjectAssign, SafeSet, StringPrototypeCharAt, StringPrototypeReplace, StringPrototypeSubstring, StringPrototypeToLowerCase, StringPrototypeToUpperCase } from "./primordials.js";
```

Comment:
Not crucial, and I'm happy to do it before landing, but `npm run format` should make everything consistent style.

Reply:
done